pre-docs/applying_manifest_blocks.md

```diff
@@ -130,24 +130,24 @@ You can optionally set these from a target's `vars`, but they don't have default
 
 Create a manifest that sets up a web server with nginx, and run it as a plan.
 
-1.  Go to the modules directory in the Bolt default Boltdir: `~/.puppetlabs/bolt/modules`
+1.  Go to the `site-modules` directory in the Bolt default project directory: `~/.puppetlabs/bolt/modules`
```

Comment:
I think that throughout this page we need to change sub `~/.puppetlabs/bolt/modules` with `~/.puppetlabs/bolt/site-modules`.